src/pages/gen2/build-a-backend/data/connect-from-server-runtime/index.mdx

```diff
@@ -169,8 +169,8 @@ export default async function handler(
   res: NextApiResponse<ResponseData>
 ) {
   const todos = await runWithAmplifyServerContext({
-    nextServerContext: { req, res}
-    operation: async (contextSpec) => {
+    nextServerContext: { req, res},
+    operation: async (contextSpec: any) => {
```

Comment:
is this `any` really needed? Shouldn't the type be inferred by TS?

Reply:
For some reason, when I first tried it I got a type error for implicit any but now I don't. Also realized that the `nextServerContext` accepts `request` and `response`, not `req` and `res`. Thanks for calling attention to this particular snippet!

Reply:
Made corrections